pglookout/pglookout.py

```diff
@@ -612,8 +623,8 @@ def check_for_maintenance_mode_file(self):
         return os.path.exists(self.config.get("maintenance_mode_file", "/tmp/pglookout_maintenance_mode_file"))
 
     def create_alert_file(self, filename):
+        filepath = os.path.join(self.config.get("alert_file_dir", os.getcwd()), filename)
```

Comment:
This is a bit dangerous:

`create_alert_file` is passed to `ClusterMonitor` which uses it outside of a try/catch through `main_monitoring_loop` => `connect_to_cluster_nodes_and_cleanup_old_nodes` => `_connect_to_db` => `create_alert_file`.

If the call to `getcwd` fails (it can raise at least `FileNotFoundError`), or the join (if the config is slightly invalid), then the `ClusterMonitor` thread will die and won't be recreated, since it's not part of a managed thread pool.

The real problem is the un-managed thread, but until we tackle that, I think it's safer to just use `filename` instead of `filepath` in the error (or have a separate try/catch when creating the `filepath` from the `filename`).

Reply:
Took the easy way out with filename